items/items-perm.md

```diff
@@ -79,15 +87,29 @@ Whether (true/false) the group is the owner of this item. Implies the maximum le
 
 ## Aggregation of permissions from multiple sources
 
-We call *aggregation* the merge of the multiple granted permissions (from possibily different groups) to one permission tuple (indexed by group, item).
+We call *aggregation* the merge of multiple granted permissions to one permission tuple `(group, item)` representing the permissions one group has on an item.
+Those granted permissions may have initially been given to different groups (the group itself or any of its ancestors).
+
+In addition, those granted permissions may have been given from multiple `source groups`,
+and come from different `origin`.
+See details in the [section on database table permissions_granted](#granted-permissions-table-permissions_granted).
+
+For each permission attribute but *can_make_session_official* and *can_enter_\** (so *can_view*, *can_grant_view*, *can_watch*, *can_edit*, and *is_owner*), we take its **maximum level** among all values. In addition, "is_owner=true" makes every other attribute get its maximum possible value.
+
+### UI for granting permissions
+
+This image shows the result of **Aggregation** for the permission `can_grant_view` given to `Group` on an `Item`:
+- **Nothing** is the permission granted directly to `Group`
+- **Solution** is the aggregated permission `Group` has, coming from the permissions given to any of its ancestors
+- Note that starting at the third level of permission, **Content**, the text is gray: it means the current-user doesn't have the right to give permissions from this point
 
-For each permission attribute but *can_make_session_official* and *can_enter_\** (so *can_view*, *can_grant_view*, *can_watch*, *can_edit*, and *is_owner*), we take its maximum level among all values. In addition, "is_owner=true" makes every other attribute get its maximum possible value.
+![UI for granting permissions]({{ site.url }}{{ site.baseurl }}/assets/screen_permissions_view.png)
 
 <a name="propagation"></a>
 
 ## Propagation across parent-child items
 
-Permissions given on an item to a group may be propagated (explicitely) to the item's children, under some conditions and depending on the permissions. This propagation computation is relaunched each time a permission is changed or an item added/removed. It is computed on the changed node (the group-item relation node) based on its parents, and then propagated to its children. When there are several parents, the higher permission among parents is kept.
+Permissions given on an item to a group may be propagated explicitly to the item's children, under some conditions and depending on the permissions. This propagation computation is relaunched each time a permission is changed or an item added/removed. It is computed on the changed node (the group-item relation node) based on its parents, and then propagated to its children. When there are several parents, the higher permission among parents is kept.
```

Comment:
The "(..)" was because it was just a repetition of what "propagated" means (it means "explicitly"). The new sentence may be understood as the "may" or "under some conditions" applying the "explicitly".